lte/cloud/go/services/subscriberdb/servicers/subscriberdb_servicer.go

```diff
@@ -30,19 +27,64 @@ import (
 	lte_protos "magma/lte/cloud/go/protos"
 	"magma/lte/cloud/go/serdes"
 	lte_models "magma/lte/cloud/go/services/lte/obsidian/models"
-	"magma/lte/cloud/go/services/subscriberdb/obsidian/models"
+	"magma/lte/cloud/go/services/subscriberdb/storage"
 	"magma/orc8r/cloud/go/services/configurator"
 	"magma/orc8r/lib/go/protos"
 )
 
 type subscriberdbServicer struct {
 	flatDigestEnabled bool
+	digestStore       storage.DigestLookup
 }
 
-const defaultSubProfile = "default"
+func NewSubscriberdbServicer(config subscriberdb.Config, digestStore storage.DigestLookup) lte_protos.SubscriberDBCloudServer {
+	servicer := &subscriberdbServicer{
+		flatDigestEnabled: config.FlatDigestEnabled,
+		digestStore:       digestStore,
+	}
+	return servicer
+}
+
+// CheckSubscribersInSync checks if the client-side subscriber digest is up-to-date with the
+// cloud version, and returns signal for whether the client needs an update.
```

Comment:
Nit here and elsewhere: generally, docstring comments for grpc methods aren't needed, since you can just reference the docstring for the grpc .proto definition